src/utils/Component/getComponent.js

```diff
@@ -47,6 +47,12 @@ const getCheckboxes = (config) => {
   return <Checkboxes {...attrs} options={options} />;
 };
 
+const getCalculation = (config) => {
+  var calculatedValue = Data.applyFormula(config);
+  const attrs = cleanAttributes(config, ['formula']);
+  return <TextInput {...attrs} value={calculatedValue} readOnly={true} />
```

Comment:
I'm surprised lint didn't complain about this.

```suggestion
  return <TextInput {...attrs} value={calculatedValue} readOnly />
```